### PR TITLE
Set allow origin header so that openapi works with localhost

### PR DESCRIPTION
**Pull request by e00E**
We're doing the same thing for the v1 price estimator.

### Test Plan
`cargo run --bin orderbook` and https://protocol-rinkeby.dev.gnosisdev.com/api/#/default/get_api_v1_orders with localhost.

### CODE REVIEW

File: orderbook/src/api.rs [hunk 1]
```diff
@@ -7,7 +7,7 @@ use warp::Filter;
 
 pub fn handle_all_routes(
     orderbook: Arc<OrderBook>,
-) -> impl Filter<Extract = impl warp::Reply, Error = warp::Rejection> + Clone {
+) -> impl Filter<Extract = (impl warp::Reply,), Error = warp::Rejection> + Clone {
```
Comment (e00E) [inline orderbook/src/api.rs:10]: Small touch up because warp expects filters to return tuples and it's needed for the `map` in main to work.

**Review by fleupold — APPROVED** [review]

**Review by josojo — APPROVED** [review]
Nice, this helps a lot!

sorry, Felix, I did not mean to re-request from you a review

**Comment by josojo** [comment]
I have a question:

if I merge all outstanding prs together - this and the renaming one -, the following command works nicely via the command line:
```
curl -X POST "http://localhost:8080/api/v1/orders" -H  "accept: application/json" -H  "Content-Type: application/json" -d "{ \"sellToken\": \"0101010101010101010101010101010101010101\",  \"buyToken\": \"0202020202020202020202020202020202020202\",  \"sellAmount\": \"42000000000000000000\",  \"buyAmount\": \"13370000000000000000\",  \"validTo\": 4294967295,  \"appData\": 0,  \"feeAmount\": \"1000000000000000000\",  \"kind\": \"sell\",  \"partiallyFillable\": false,   \"signature\": \"1b97be7a77d916c99b39d2909c5fde76a82f4fd49868b18d000d93de2c5906160268f04221fd6ac0e8aebfb048cf6c75ca882cd20c6e06aec094c278f8e44c1759\"}" 
{"UID":"ae330dfc6531ddb8c3ef55c80ff41ef4cced2af4f3c6bb0f8c727d589be9886af339ab35c77dc6117c556d57a6336abeca34596affffffff"}%        
```
But, swagger.editor keeps on telling me:
```

Undocumented | TypeError: Failed to fetch
```
Do you guys have the same issue?

**Comment by fleupold** [comment]
I'm getting 

> Request body deserialize error: missing field `tip` at line 1 column 433

**Comment by josojo** [comment]
Because you are running against rinkeby. if you run it against local-host, this should no longer be the case with the code updates.

**Comment by fleupold** [comment]
I was expecting autodeploy to automatically push those changes. But In fact, the pod is currently using the wrong image tag and thus it was restarting always with the old version. I will fix that right away.

**Comment by fleupold** [comment]
Works now! 🎉 

image

**Pull request merged**